### PR TITLE
Add GitHub templates, update license, and remove unused files

### PR DESCRIPTION
**Pull request by RobertWSaunders**
## Proposed Change

This adds GitHub templates to the project updates the license with the correct year and removes some unused files. The templates are good to keep things consistent when we submit a pull request or issue. See the detailed list of changes below.

I've also made myself a code owner so every PR made to this repo will require my approval. I'm open to making others code owners if they have enough knowledge of the codebase. 

 ### How did you do this?

Adding new files and making updates to existing files!

 ### Why are you choosing this approach?

Keep things up to date and improve developer experience.

 ### Any open questions you want to ask code reviewers?

Nope! 😄 

 ### List of changes:

  - Add GitHub templates
    - Pull request template
    - Issue templates
    - Code owners document
   - Update license and add license file
   - Remove Procfile (Heroku related)
   - Remove db schema (dated)

 ## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new tests for my core changes (where applicable).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
@maxkaran @CarsonCook I made a few repository config changes as well. See below and let me know what you think:

- Removed stale branches, only two branches now are `master` and `dev`.
- I made `dev` the default branch for the repo, with the intention that `master` is production.
  - Any work can be merged into `dev` through pull requests, when we are ready to move those changes into production we can make a pull request to merge `dev` into `master` and run deploy once the changes are merged.
  - Pull requests into `dev` require one approval and at least one approval from a code owner (me).
  - Pull requests into `master` require two approvals and at least one approval from a code owner.
- Anything that is committed to `dev` or `master` needs a pull request made.

I'll try to make updates to this repo to clean it up a bit, it can be your discretion if you want to pipe those changes into your fork of the project. Nonetheless I'll be tagging you guys in all the pull requests I make and will expect you guys to review them for me.

**Comment by RobertWSaunders** [comment]
I also made the Travis CI status check to be required before merging. i.e. tests need to pass before anything can be merged. Do you guys agree with this?

**Review by maxkaran — APPROVED** [review]

**Pull request merged**
